data/language/english_uk.txt

```diff
@@ -4111,6 +4111,19 @@ STR_5802    :{SMALLFONT}{BLACK}Stops guests from littering and vomiting
 STR_5803    :{SMALLFONT}{BLACK}Rotate selected map element
 STR_5804    :Mute sound
 
+#tooltips
+STR_5805    :{SMALLFONT}{BLACK}Use GPU for graphics instead of the CPU
```

Comment:
"Use GPU for displaying instead of CPU. Improves compatibility with screen capture software. May slightly decrease performance."